data/raw_data_storage.json

```diff
@@ -1,3 +1,10980 @@
 {
-
-}
+    "RAW_DATA": [
```

Comment:
This file is meant to stay empty and store data only after the tool is run. Please make sure to keep it empty similar to how it is in the master branch with the two curly braces. Also please remember to discard changes made to this file after running the tool

Reply:
I cleared up this file and left 
```
{
}
```